docs/adr/files/0009/option-b-android-components.diff

```diff
@@ -0,0 +1,256 @@
+diff --git a/mobile/android/android-components/components/feature/fxsuggest/build.gradle b/mobile/android/android-components/components/feature/fxsuggest/build.gradle
+index d5b717c0ef..28d9c82eb2 100644
```

Comment:
I wasn't sure the best way to communicate these changes, but I went with checking in diffs because I wanted to make sure they never expired.  I can put these into a github PR or phab patch if people want, maybe those would look nicer.